### PR TITLE
Markupsafe update and point to Nisaba as main Oracc edition tool

### PR DESCRIPTION
**Pull request by raquel-ucl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,22 @@
 [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.1414437.svg)](https://doi.org/10.5281/zenodo.1414437)
 
-# Nammu: Desktop GUI for ORACC
+
+
+| Please note that as of September 2021, the recommended desktop tool for Oracc edition is [Nisaba](https://github.com/oracc/nisaba), an extension for Visual Studio Code. |
+|-|
```
Comment (ageorgou) [inline README.md:6]: I think this is obvious enough!

**Review by ageorgou — APPROVED** [review]
All looks good, but should we wait until Nisaba is published and can be installed? Although having the warning already may be useful.

**Comment by raquel-ucl** [comment]
We can wait to merge this until Nisaba is released.

**Comment by ageorgou** [comment]
Oh, right :) We might as well merge it now then, I guess.

**Pull request merged**